pallets/automation-time/src/lib.rs

```diff
@@ -515,18 +515,19 @@ pub mod pallet {
 		/// * `DuplicateTask`: There can be no duplicate tasks.
 		/// * `TimeSlotFull`: Time slot is full. No more tasks can be scheduled for this time.
 		/// * `TimeTooFarOut`: Execution time or frequency are past the max time horizon.
+		// TODO: needs real benchmarking
```

Comment:
Will follow up in separate PR. This is unrelated work that I noticed while doing this.